scripts/prepare_sector_network.py

```diff
@@ -3801,26 +3805,28 @@ def add_geothermal_district_heating_supply(n, egs_potential):
 
     # import sys
     # sys.exit()
-
-    for potential, mode in [
-        (snakemake.input["egs_potentials_egs"], "egs"),
-        (snakemake.input["egs_potentials_hs"], "hs"),
-    ]:
-        attach_enhanced_geothermal(n, potential, mode)
+    #breakpoint()
+    if not "noEGS" in snakemake.wildcards.sopts:
+        for potential, mode in [
+            (snakemake.input["egs_potentials_egs"], "egs"),
+            (snakemake.input["egs_potentials_hs"], "hs"),
+        ]:
+            attach_enhanced_geothermal(n, potential, mode)
 
     industry_demands = pd.read_csv(
         snakemake.input["industrial_heating_demands"], index_col=0
     )
 
     logger.info("Adding industrial heating demands.")
     add_industry_demand(n, industry_demands)
+    
+    if not "noEGS" in snakemake.wildcards.sopts:
+        industry_egs_supply = pd.read_csv(
+            snakemake.input["industrial_heating_egs_supply_curves"], index_col=[0, 1]
+        )
 
-    industry_egs_supply = pd.read_csv(
-        snakemake.input["industrial_heating_egs_supply_curves"], index_col=[0, 1]
-    )
-
-    logger.info("Adding geothermal supply for industry.")
-    add_geothermal_industry_supply(n, industry_egs_supply)
+        logger.info("Adding geothermal supply for industry.")
+        add_geothermal_industry_supply(n, industry_egs_supply)

```

Comment:
As a side note, we may be missing currently the resistive heaters as a possible source of the heat. Not sure what will be the implications for the system under the scenarios without geothermal. Worth to check